external/llvm-project/mlir/include/mlir/Dialect/Async/Passes.h

```diff
@@ -31,7 +31,8 @@ std::unique_ptr<Pass> createAsyncParallelForPass(bool asyncDispatch,
                                                  int32_t numWorkerThreads,
                                                  int32_t minTaskSize);
 
-std::unique_ptr<OperationPass<ModuleOp>> createAsyncToAsyncRuntimePass();
+std::unique_ptr<OperationPass<ModuleOp>>
+createAsyncToAsyncRuntimePass(bool enableCoroutines = true);
```

Comment:
Instead of doing this, use the pass option struct that gets automatically generated now

Reply:
You could also, probably, remove the `let constructor = ...` entirely and, if you wanted to keep back-compat, rename the tablegen definition to `AsyncToAsyncRuntimePass`